Jenkinsfile.ci

```diff
@@ -338,6 +341,21 @@ def linuxPythonTesting(env_name, network_name, testEnv) {
     }
 }
 
+def linuxNodejsTesting(env_name, network_name, testEnv) {
+    unstash name: "LibindyNodejsSO${env_name}"
+    dir('wrappers/nodejs') {
+        testEnv.inside("--network=${network_name}") {
+            echo "${env_name} Libindy Test: Test nodejs wrapper"
+
+            sh '''
+                npm run cp-include
```

Comment:
I added in ubuntu.dockerfile the following lines:
```
RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
RUN apt-get install -y nodejs
RUN npm install -g node-gyp
```
I am getting ```missing script: cp-include```  on executing this command ```npm run cp-include```